apps/frontend/core-dapp/src/stores/graphs/UniswapV3GraphStore.ts

```diff
@@ -78,7 +83,12 @@ export class UniswapV3GraphStore extends GraphStore<RootStore, SupportedContract
         maxDataPerFetch: Math.min(days, UNISWAP_MAX_FIRST),
         onNewData: handleNewHistoricalPoolData(days),
       }
-    )?.data
+    )
+
+    runInAction(() => {
+      if (query?.error) this.outOfService = true
```

Comment:
```suggestion
      this.outOfService = !!query?.error
```

Otherwise, our chart will be disabled until the user refreshes the app if a query fails for any reason (i.e. no internet), right?

Reply:
ahh yes good point